### PR TITLE
Jordeen/heroku setup

### PR DESCRIPTION
**Pull request by jodreen**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -81,5 +77,4 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  config.action_mailer.default_url_options = { host: 'please-change-me.com' }
```
Comment (SamLau95) [inline config/environments/production.rb]: Did you mean to remove this? You'll need it for emailing.

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -1,298 +0,0 @@
...
-  #
-  #     mount MyEngine, at: '/my_engine'
-  #
-  # The router that invoked `devise_for`, in the example above, would be:
-  # config.router_name = :my_engine
-  #
-  # When using omniauth, Devise cannot automatically set Omniauth path,
-  # so you need to do it manually. For the users scope, it would be:
-  # config.omniauth_path_prefix = '/my_engine/users/auth'
-end
```
Comment (SamLau95) [inline config/initializers/devise.rb]: You don't need authentication anymore?

File: config/environments/production.rb (hunk 1 above)
Comment (jodreen) [inline config/environments/production.rb]: we don't have a use for emailing actually!

File: config/initializers/devise.rb (hunk 2 above)
Comment (jodreen) [inline config/initializers/devise.rb]: @SamLau95 we don't have users actually it's just a password

Comment (SamLau95) [inline config/initializers/devise.rb]: Oh haha chill!

File: config/environments/production.rb (hunk 1 above)
Comment (SamLau95) [inline config/environments/production.rb]: Ok cool! You should also make the relevant changes to `staging.rb` then. And you won't need the `letter_opener` and `recipient_interceptor` gems.

Comment (jodreen) [inline config/environments/production.rb]: Thanks! Just fixed everything

**Pull request closed without merging**